trl/trainer/dpo_trainer.py

```diff
@@ -422,13 +422,13 @@ def dpo_loss(
         reference_free: bool = False,
     ) -> Tuple[torch.FloatTensor, torch.FloatTensor, torch.FloatTensor]:
         """Compute the DPO loss for a batch of policy and reference model log probabilities.
+        The beta is a temperature parameter for the DPO loss, typically something in the range of 0.1 to 0.5. We ignore the reference model as beta -> 0.
```

Comment:
This comment is a bit cryptic since `beta` isn't part of the args. Would it be better to move this as a comment directly above the code, e.g. here: https://github.com/huggingface/trl/pull/1022/files#diff-5bbdb5d54108f2162b47bc54dc23c7b8e7744d2941118e60a44c161a4acc0ee8L454-L455